config/settings.py

```diff
@@ -35,6 +35,7 @@
 
 # Domain of the new frontend (it is used to redirect the user to new pages not implemented in Django)
 NEW_FRONTEND_DOMAIN = os.environ.get("NEW_FRONTEND_DOMAIN")
+NEW_FRONTEND_URL = os.environ.get("NEW_FRONTEND_URL")
```

Comment:
If you have the domain, I don't think you need this.

Reply:
My concern is with the domain only I cannot have the used scheme.